SPEC.md

```diff
@@ -146,48 +147,61 @@ Plugins that consume any of these configuration keys should respect their intend
 Plugins may define additional fields that they accept and may generate an error if called with unknown fields. Runtimes must preserve unknown fields in plugin configuration objects when transforming for execution.
 
 #### Example configuration
+
+`/etc/cni/net.d/10-dbnet.conf`:
 ```jsonc
 {
   "cniVersion": "1.1.0",
   "cniVersions": ["0.3.1", "0.4.0", "1.0.0", "1.1.0"],
   "name": "dbnet",
-  "plugins": [
-    {
-      "type": "bridge",
-      // plugin specific parameters
-      "bridge": "cni0",
-      "keyA": ["some more", "plugin specific", "configuration"],
-      
-      "ipam": {
-        "type": "host-local",
-        // ipam specific
-        "subnet": "10.1.0.0/16",
-        "gateway": "10.1.0.1",
-        "routes": [
-            {"dst": "0.0.0.0/0"}
-        ]
-      },
-      "dns": {
-        "nameservers": [ "10.1.0.1" ]
-      }
-    },
-    {
-      "type": "tuning",
-      "capabilities": {
-        "mac": true
-      },
-      "sysctl": {
-        "net.core.somaxconn": "500"
-      }
-    },
-    {
-        "type": "portmap",
-        "capabilities": {"portMappings": true}
-    }
-  ]
```

Comment:
(I already mentioned in meeting, but add it for further discussion)

Is your PR completely changes the current style (deprecate current single JSON config) and introduce new one to replace? or Introduce new one additionally which co-exists with current style?